controller/network/route_perf_test.go

```diff
@@ -0,0 +1,57 @@
+package network
+
+import (
+	"fmt"
+	"github.com/openziti/fabric/controller/db"
+	"github.com/openziti/foundation/identity/identity"
+	"github.com/openziti/foundation/transport/tcp"
+	"testing"
+)
+
+func BenchmarkShortestPathPerf(b *testing.B) {
```

Comment:
are there also correctness tests that are run? as someone not deeply familiar with these tests i'd feel better knowing those were there too :)

Reply:
Added correctness test to ensure new behavior matches old behavior